Jenkinsfile

```diff
@@ -0,0 +1,25 @@
+pipeline {
```

Comment:
syntax is good, logic and indents are not:
- Build step deploys in fact
- scp using root
- archive after deploy is barely needed